airbyte-ci/connectors/metadata_service/lib/metadata_service/models/generated/ConnectorMetrics.py

```diff
@@ -0,0 +1,24 @@
+# generated by datamodel-codegen:
+#   filename:  ConnectorMetrics.yaml
+
+from __future__ import annotations
+
+from typing import Any, Optional
+
+from pydantic import BaseModel, Extra
+from typing_extensions import Literal
+
+
+class ConnectorMetrics(BaseModel):
+    all: Optional[Any] = None
+    cloud: Optional[Any] = None
+    oss: Optional[Any] = None
+
+
+class ConnectorMetric(BaseModel):
+    class Config:
+        extra = Extra.allow
+
+    usage: Optional[Literal["none", "low", "medium", "high"]] = None
+    sync_success_rate: Optional[Literal["none", "low", "medium", "high"]] = None
```

Comment:
`"none"` as a valid value, versus None as a valid param since the value itself is optional, feels easy to mess up to me. Maybe `no_usage` and `not_applicable` for usage and sync success rate would be clearer? (is 0% success rate "low", or "none"? i'm assuming the former)

Reply:
Good call

Reply:
Passing that through to Alex Gron.